data-prepper-api/src/test/java/org/opensearch/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -456,6 +456,33 @@ public void testBuild_withStringData() {
         assertThat(event.get("foo", String.class), is(equalTo("bar")));
     }
 
+    @Test
+    public void testBuild_withFormatString() {
```

Comment:
Also, there should be some test cases for other edge cases - The value is `null`. The value is an empty string.

Reply:
I have added the edge test cases and it looks like all tests are passing now.